rootfs/usr/share/www/index.html

```diff
@@ -5,7 +5,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta charset="utf-8" />
   <title>Preparing Home Assistant</title>
+  <link rel="shortcut icon" href="/landingpage/static/favicon.ico" />
   <link rel="icon" type="image/svg+xml" href="/landingpage/static/logo.svg" />
+  <link rel="mask-icon" href="/landingpage/static/mask-icon.svg" color="#1bbdf2">
```

Comment:
Not sure if it matters with nowadays HTML

```suggestion
  <link rel="mask-icon" href="/landingpage/static/mask-icon.svg" color="#1bbdf2" />
```